samples/cpp/classification_sample_async/main.cpp

```diff
@@ -84,37 +84,62 @@ int main(int argc, char* argv[]) {
 
         // -------- Step 2. Read a model --------
         slog::info << "Loading model files:" << slog::endl << FLAGS_m << slog::endl;
-        std::shared_ptr<ov::Model> model = core.read_model(FLAGS_m);
-        printInputAndOutputsInfo(*model);
-
-        OPENVINO_ASSERT(model->inputs().size() == 1, "Sample supports models with 1 input only");
-        OPENVINO_ASSERT(model->outputs().size() == 1, "Sample supports models with 1 output only");
-
-        // -------- Step 3. Configure preprocessing --------
-        const ov::Layout tensor_layout{"NHWC"};
-
-        ov::preprocess::PrePostProcessor ppp(model);
-        // 1) input() with no args assumes a model has a single input
-        ov::preprocess::InputInfo& input_info = ppp.input();
-        // 2) Set input tensor information:
-        // - precision of tensor is supposed to be 'u8'
-        // - layout of data is 'NHWC'
-        input_info.tensor().set_element_type(ov::element::u8).set_layout(tensor_layout);
-        // 3) Here we suppose model has 'NCHW' layout for input
-        input_info.model().set_layout("NCHW");
-        // 4) output() with no args assumes a model has a single result
-        // - output() with no args assumes a model has a single result
-        // - precision of tensor is supposed to be 'f32'
-        ppp.output().tensor().set_element_type(ov::element::f32);
-
-        // 5) Once the build() method is called, the pre(post)processing steps
-        // for layout and precision conversions are inserted automatically
-        model = ppp.build();
+        bool isNetworkCompiled = fileExt(FLAGS_m) == "blob";
+
+        ov::CompiledModel compiledModel;
+        const ov::Layout tensor_layout{ "NHWC" }; 
+
+        if (!isNetworkCompiled)
```

Comment:
To follow the spirit of https://github.com/openvinotoolkit/openvino/pull/9463 , I think we need to keep the naming convention the same as other example codes (snake case) although OpenVINO itself uses camel case for "old" API and snake case for OV API 2.0.  So is keeping "{" in the same line.

Reply:
Thanks @peterchen-intel, please take a look at latest set of changes.